packages/mco/components/discovered-application-wizard/wizard-steps/namespace-step/namespace-table.tsx

```diff
@@ -178,16 +208,13 @@ export const NamespaceSelectionTable: React.FC<NamespaceSelectionTableProps> =
             setSelectedRows={setSelectedNamespaces}
             loaded={searchLoaded}
             borders={false}
-            emptyRowMessage={() => (
-              <Bullseye className="mco-namespace-selection__border pf-v5-u-mt-md pf-v5-u-mb-sm pf-v5-u-h-33vh">
-                {!!searchError ||
-                  (!clusterName
-                    ? t(
-                        'There are no namespaces to display. Select a cluster first to view namespaces.'
-                      )
-                    : t('There are no namespaces to display.'))}
-              </Bullseye>
-            )}
+            emptyRowMessage={
+              searchError
+                ? null
+                : clusterName
+                ? EmptyRowMessage
+                : NoClusterEmptyRowMessage
```

Comment:
```suggestion
              searchError
                ? searchError
                : clusterName
                ? EmptyRowMessage
                : NoClusterEmptyRowMessage
```

if a search error is found, we need to display searchError.

Reply:
Done.